INSTALL.md

```diff
@@ -0,0 +1,18 @@
+# Installing Cantera
+
+## Basic Instructions
+
+To compile using the default options, run `scons build` followed by
+`scons install`.
+
+Configuration options are specified with `name=value` on the command line, e.g.:
+`scons build optimize=n prefix=/home/$USER/cantera`
+
+The full list of configuration options and their default values can be shown by running
+`scons help --options`. The list of available scons commands (e.g. `build`) can be
```

Comment:
In this case, I'm tempted to go with changing this to ``` `scons` ``` (with the backticks) since these are specifically options to the command line tool `scons`, which is spelled with all lowercase.

Reply:
Sounds good to me